fast-vm

```diff
@@ -966,9 +982,38 @@ case "$1" in
 				pmsg $P_ERROR "$3 not executable\n" $vm_number
 			fi
 		else
-			ssh -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "root@192.168.$SUBNET_NUMBER.$vm_number"
+			use_sshkey_if_exists
+			ssh $FASTVM_USE_SSHKEY -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "root@192.168.$SUBNET_NUMBER.$vm_number"
 		fi
 		;;
+	scp|keydist)
+		vm_number="$2"
+		if [ -z "$vm_number" ]; then pmsg $P_ERROR "VM number missing\n"; usage ssh; fi
+		if [[ $vm_number = *[\ ]* ]]; then handle_multiple_requests $@;	fi
+
+		check_vm_number "$vm_number" "inactive"
+		update_access_time "$vm_number"
+
+		wait_for_ssh "$vm_number"
+		if [ "$1" == "keydist" ]; then
+			ssh-copy-id -i "$FASTVM_USER_CONF_DIR/id_fastvm" \
+				-o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
+				"root@192.168.$SUBNET_NUMBER.$vm_number" | sed "s/with:.*/with: $0 ssh $vm_number/"
```

Comment:
What about filtering out more outputs and leave just 2 most important things - 'prompt for password' and 'information what command to use next'?
~~~
ssh-copy-id -i "$FASTVM_USER_CONF_DIR/id_fastvm" \
				-o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
				"root@192.168.$SUBNET_NUMBER.$vm_number" 2>&1 |egrep '(password|Now try logging into the machine)'| sed "s/with:.*/with: $0 ssh $vm_number/"
~~~

Reply:
Sure!  I changed the delimiter in sed to '@' to avoid problems with slashes in $0.

Please let me know what you think of the latest version.  I've updated the manual page as well.  Thank you!